xml/html_title.xml

```diff
@@ -3247,4 +3391,332 @@
+    <param pos="0" name="hw.vendor" value="SpotterRF"/>
+    <param pos="0" name="hw.device" value="Sensor"/>
+    <param pos="0" name="hw.product" value="Drone Detector"/>
+  </fingerprint>
+
+  <fingerprint pattern="^Polycom (DMA \d+)$">
+    <description>Polycom DMA 7000</description>
+    <example hw.product="DMA 7000">Polycom DMA 7000</example>
+    <param pos="0" name="hw.vendor" value="Polycom"/>
+    <param pos="0" name="hw.device" value="VoIP"/>
+    <param pos="1" name="hw.product"/>
+  </fingerprint>
+
+  <fingerprint pattern="^VVX(\d{3})$">
+    <description>Polycom VVX VoIP Phones</description>
+    <example hw.model="250">VVX250</example>
+    <param pos="0" name="hw.vendor" value="Polycom"/>
+    <param pos="0" name="hw.device" value="VoIP"/>
+    <param pos="1" name="hw.model"/>
+    <param pos="0" name="hw.product" value="VVX {hw.model}"/>
+  </fingerprint>
+
+  <fingerprint pattern="^(OBi[23]\d{2})">
+    <description>Poly OBI Analog Telephone Adapter</description>
+    <example hw.product="OBi200">OBi200</example>
+    <param pos="0" name="hw.vendor" value="Poly"/>
+    <param pos="0" name="hw.device" value="VoIP"/>
+    <param pos="1" name="hw.product"/>
+  </fingerprint>
+
+  <fingerprint pattern="^Alpha Technologies$">
+    <description>Alpha Technologies Novus UPS</description>
+    <example>Alpha Technologies</example>
+    <param pos="0" name="hw.vendor" value="Alpha Technologies"/>
+    <param pos="0" name="hw.device" value="Power Device"/>
+    <param pos="0" name="hw.product" value="Novus UPS"/>
+  </fingerprint>
+
+  <fingerprint pattern="^Network Management Card$">
+    <description>Eaton UPS Network Card</description>
+    <example>Network Management Card</example>
+    <param pos="0" name="hw.vendor" value="Eaton"/>
+    <param pos="0" name="hw.device" value="Power Device"/>
+    <param pos="0" name="hw.product" value="UPS"/>
+    <param pos="0" name="hw.certainty" value="0.5"/>
+  </fingerprint>
+
+  <fingerprint pattern="^UPS Properties$">
+    <description>APC UPS Network Card</description>
+    <example>UPS Properties</example>
+    <param pos="0" name="hw.vendor" value="APC"/>
+    <param pos="0" name="hw.device" value="Power Device"/>
+    <param pos="0" name="hw.product" value="UPS"/>
+    <param pos="0" name="hw.certainty" value="0.5"/>
+  </fingerprint>
+
+  <fingerprint pattern="^HP ([\w ]+) PostScript">
+    <description>HP Printer</description>
+    <example hw.product="Designjet T1200">HP Designjet T1200 PostScript - Supplies</example>
+    <param pos="0" name="hw.vendor" value="HP"/>
+    <param pos="0" name="hw.family" value="JetDirect"/>
+    <param pos="0" name="hw.device" value="Printer"/>
+    <param pos="1" name="hw.product"/>
+  </fingerprint>
+
+ <fingerprint pattern="^Radio Configuration/Diagnostic Utility$">
+    <description>ProSoft RadioLinx</description>
+    <example>Radio Configuration/Diagnostic Utility</example>
+    <param pos="0" name="hw.vendor" value="ProSoft"/>
```

Comment:
The vendor here should probably be `Prosoft Technology`. We will likely end up creating a CPE remap to add the dash.

```
a:prosoft-technology:radiolinx_controlscape
h:prosoft-technology:icx35-hwc-a
h:prosoft-technology:icx35-hwc-e
o:prosoft-technology:icx35-hwc-a_firmware
o:prosoft-technology:icx35-hwc-e_firmware
```